1.6.0-rc1/java compile error

make failure on standalone machine with java enabled, out-of-tree-build:

```shell
Making install in bindings/java/src/main/native
make[1]: Entering directory `/usr/local/src/ucx/build/bindings/java/src/main/native'
mvn -B -f ../../../../../../ucx-1.6.0/bindings/java/pom.xml -Dmaven.repo.local=/usr/local/src/ucx/build/.deps/ compile native:javah
POM file ../../../../../../ucx-1.6.0/bindings/java/pom.xml specified with the -f/--file command line argument does not exist
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-readable POM /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0/bindings/java/pom.xml: /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0/bindings/java/pom.xml (No such file or directory) @
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project  (/usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0/bindings/java/pom.xml) has 1 error
[ERROR]     Non-readable POM /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0/bindings/java/pom.xml: /usr/local/src/ucx/build/bindings/java/src/main/native/../../../../../../ucx-1.6.0/bindings/java/pom.xml (No such file or directory)
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
make[1]: *** [org_ucx_jucx_ucp_UcpContext.h] Error 1
make[1]: Leaving directory `/usr/local/src/ucx/build/bindings/java/src/main/native'
make: *** [install-recursive] Error 1
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -47,6 +47,7 @@ endif
 
 if HAVE_JAVA
 SUBDIRS += bindings/java/src/main/native
+EXTRA_DIST += bindings/java
 endif
 
 EXTRA_DIST += contrib/configure-devel

